adduser.py

```diff
@@ -30,7 +32,7 @@ def read_user_data():
 
 def verify_phone_number(phone):
     return phone.startswith('+') \
-        and len(phone) == 13 \
+        and len(phone) >= 8 \
```

Comment:
99% of phone numbers are 13 characters long, which is why we should display a warning if the user inputs a phone number with a different length (far more likely to happen because of a typo than the phone number actually being a different length). The user can still add the number, even if a warning is displayed

```suggestion
        and len(phone) == 13 \
```